Mathlib/Geometry/Manifold/ContMDiff/Basic.lean

```diff
@@ -272,20 +272,26 @@ theorem contMDiff_of_mulTSupport [One M'] {f : M → M'}
   by_cases hx : x ∈ mulTSupport f
   · exact hf x hx
   · exact ContMDiffAt.congr_of_eventuallyEq contMDiffAt_const
-      (not_mem_mulTSupport_iff_eventuallyEq.1 hx)
+      (notMem_mulTSupport_iff_eventuallyEq.1 hx)
 
-@[to_additive contMDiffWithinAt_of_not_mem]
-theorem contMDiffWithinAt_of_not_mem_mulTSupport {f : M → M'} [One M'] {x : M}
+@[to_additive contMDiffWithinAt_of_notMem]
+theorem contMDiffWithinAt_of_notMem_mulTSupport {f : M → M'} [One M'] {x : M}
     (hx : x ∉ mulTSupport f) (n : WithTop ℕ∞) (s : Set M) : ContMDiffWithinAt I I' n f s x := by
   apply contMDiffWithinAt_const.congr_of_eventuallyEq
-    (eventually_nhdsWithin_of_eventually_nhds <| not_mem_mulTSupport_iff_eventuallyEq.mp hx)
-    (image_eq_one_of_nmem_mulTSupport hx)
+    (eventually_nhdsWithin_of_eventually_nhds <| notMem_mulTSupport_iff_eventuallyEq.mp hx)
+    (image_eq_one_of_notMem_mulTSupport hx)
+
+@[deprecated (since := "2025-05-23")]
```

Comment:
the to_additive version is missing a deprecation. Same thing for the next lemma.

Reply:
I've fixed here and everywhere by printing all non-auto declarations on `master` and this branch, then comparing these files.